src/ecmascript_simd.js

```diff
@@ -34,10 +34,9 @@
 (function(global) {
 
 if (typeof global.SIMD === "undefined") {
-  // SIMD module. We don't use the var keyword here, so that we put the
-  // SIMD object in the global scope even if this polyfill code is included
-  // within some other scope. The theory is that we're anticipating a
-  // future where SIMD is predefined in the global scope.
+  // SIMD module. We put the SIMD object in the global scope, because the theory
```

Comment:
This seems unrelated to the change description. Please remove it and submit a comment update as a separate pull request (with justification).